perfkitbenchmarker/benchmark_spec.py

```diff
@@ -65,7 +68,12 @@
     DIGITALOCEAN: {
         IMAGE: 'ubuntu-14-04-x64',
         MACHINE_TYPE: '2gb',
-        ZONE: 'sfo1'
+        ZONE: 'sfo1',
+    },
+    RACKSPACE: {
```

Comment:
Current 'dev' no longer uses the DEFAULTS dict as of https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/commit/c7233241ce41c9e629a9046ce2b40d4288ad7896 , the defaults are now provided by the VM subclasses. Can you follow the digitalocean example in that commit to move the defaults?